jwst/regtest/test_associations_sdp_pools.py

```diff
@@ -54,6 +54,48 @@ def _assoc_sdp_against_standard(rtdata, resource_tracker, request, pool_args):
     compare_asn_files(out_paths, truth_paths)
 
 
+# NOTE: These are inflight equivalent approximate replacements for
+#       test_associations_standards.py test module.
+@pytest.mark.parametrize(
+    "pool_args",
+    [
+        ("jw05204_20250308t202944_pool", []),  # pool_002_image_miri
```

Comment:
Note to self: When things are done, I want to reorder alphabetically by the SDP pools rather than old pool names in comments, to ensure we do not accidentally parametrize the same SDP pool twice.